fix getNode error

abiii-daniele opened the pull request.

abiii-daniele committed:
fix getNode error

mosenco reviewed (changes requested):
using try except is recommended for testing only. I suggest to substitute `badChannelList.setCurrentNode(slicer.util.getNode('recon'))`
into
`badChannelList.setCurrentNode(slicer.mrmlScene.GetFirstNodeByName('recon'))`

ref: [https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide#Slicer_4.9:_slicer.util.getNode.28.29_raises_exception_if_node_not_found](https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide#Slicer_4.9:_slicer.util.getNode.28.29_raises_exception_if_node_not_found)

mosenco committed:
slicer.util.getNode() cant find node

solved substituting slicer.util.getNode with slicer.mrmlScene.GetFirstNodeByName()
